backend/database/migrations/2022_11_16_134040_create_questions_table.php

```diff
@@ -21,7 +21,8 @@ public function up()
             $table->string("correct_answers");
             $table->boolean("is_multiple_answers");
             $table->string('category_id');
-            $table->string('assessment_id');
+            $table->string('assessment_id'); 
+            $table->string('company_id');
```

Comment:
company_id is a new introduction. It should also be done using update migration method